awx/ui/src/screens/Job/JobOutput/getLineTextHtml.js

```diff
@@ -0,0 +1,107 @@
+import Ansi from 'ansi-to-html';
+import hasAnsi from 'has-ansi';
+import { encode } from 'html-entities';
+
+const EVENT_START_TASK = 'playbook_on_task_start';
+const EVENT_START_PLAY = 'playbook_on_play_start';
+const EVENT_STATS_PLAY = 'playbook_on_stats';
+const TIME_EVENTS = [EVENT_START_TASK, EVENT_START_PLAY, EVENT_STATS_PLAY];
+
+const ansi = new Ansi({
+  stream: true,
+  colors: {
+    0: '#000',
+    1: '#A30000',
+    2: '#486B00',
+    3: '#795600',
+    4: '#00A',
+    5: '#A0A',
+    6: '#004368',
+    7: '#AAA',
+    8: '#555',
+    9: '#F55',
+    10: '#5F5',
+    11: '#FF5',
+    12: '#55F',
+    13: '#F5F',
+    14: '#5FF',
+    15: '#FFF',
+  },
+});
+
+function getTimestamp({ created }) {
+  const date = new Date(created);
```

Comment:
Suggestion: use luxon here as we now have it at our disposal.

Reply:
Oh good catch. I'll try and catch than in the next PR, since I'm diving back into this component :)